packages/cz-lerna-changelog/src/index.ts

```diff
@@ -0,0 +1,131 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2015 - present Instructure, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore not typed
+import commitAnalyzer from '@semantic-release/commit-analyzer'
+import chalk from 'chalk'
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore not typed
+import buildCommit from 'cz-customizable/lib/build-commit'
+import autocomplete from 'inquirer-autocomplete-prompt'
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore not typed
+import { getChangedPackages, getPackages } from '@instructure/pkg-utils'
+import { makeDefaultQuestions } from './make-default-questions'
+import { autoCompleteQuestions } from './autocomplete-questions'
+
```

Comment:
typing is far from perfect here, the goal was to get rid of commonJS imports instead of perfect typing